torchbenchmark/util/framework/gnn/model_factory.py

```diff
@@ -88,6 +88,13 @@ def __init__(self, model_name, test, device, jit=False, batch_size = None, extra
     def get_module(self):
         return self.model, self.example_inputs[0]
 
+    def enable_bf16(self):
```

Comment:
Why we still need this override function in gnn model factory?  I think the example input conversion also can be covered by the new `input_convert` function in model.py. We may double check it

Reply:
The input of gnn model is {x, edge_index}, edge_index is `int64` tensor, which is not converted to `bfloat16`.

Reply:
ok, we can modify the `input_convert` to check the origin tensor datatype, if original is `float32` then we do the conversion. How do you think?

Reply:
Sure. Done.

Reply:
OK, we should able to delete this `enable_bf16` function in gnn model factory safely now.